### ISSUE TITLE
Create a comparison table of features between r2 vs cutter

### ISSUE THREAD

**Issue by radare** [radareorg/cutter#1283]

**Comment by ITAYC0HEN** [comment]
Why is this important? We have an organized roadmap for months, and ideas for years ahead. And we implement things and invest our time by walking this roadmap.

We are structuring the future capabilities of Cutter by our users' requests and by our vision of how Cutter should look like. 

radare2 is super powerful, it has tons of features which some are must have in Cutter and some absolutely not. We shouldn't compare what Cutter is doing to what radare2 can do. Luckily, radare2 allows us to be very flexible and has most of the things we need in its core.

Bottom line, radare2 have so much features which we don't want to support in Cutter, and this is why we are going by a roadmap of features instead of trying to narrow the distance from radare2's ability.

**Comment by xarkes** [comment]
Agreed, I think this can be closed for now :)
